### PR TITLE
Animation backend code

### PR DESCRIPTION
**Pull request by LeoWikstrom**
The animation code from last sprint

### CODE REVIEW

**Review by fredrix96 — DISMISSED** [review]
Code seems fine and the animation is displayed as it should, however, the shop scene in Sandbox crashes

**Review by fredrix96 — DISMISSED** [review]
Seems fine. Good job!

File: StortSpelProjekt/Engine/src/Headers/structs.h [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 #define STRUCTS_H
 
+// Light defines
 #define MAX_DIR_LIGHTS   10
 #define MAX_POINT_LIGHTS 10
 #define MAX_SPOT_LIGHTS  10
 
+// Animation defines
+#define MAX_BONES_PER_VERTEX 10
+#define MAX_ANIMATION_MATRICES 52
```
Comment (jocke1995) [inline StortSpelProjekt/Engine/src/Headers/structs.h]: Should not be 52

**Review by jocke1995 — CHANGES_REQUESTED** [review]
Minor change that you need to fix

File: StortSpelProjekt/Sandbox/src/main.cpp [hunk 2]
```diff
@@ -14,6 +14,7 @@ Scene* WilliamsTestScene(SceneManager* sm);
 Scene* AndresTestScene(SceneManager* sm);
 Scene* AntonTestScene(SceneManager* sm);
 Scene* BjornsTestScene(SceneManager* sm);
+Scene* ShopScene(SceneManager* sm);
```
Comment (jocke1995) [inline StortSpelProjekt/Sandbox/src/main.cpp]: Why did you add this again? :/ Remove from sandbox

**Review by jocke1995 — CHANGES_REQUESTED** [review]
Change

**Review by jocke1995 — APPROVED** [review]

**Review by fredrix96 — APPROVED** [review]
Looks good, good work!

**Pull request merged**